plugins/pulp_rpm/plugins/importers/yum/upload.py

```diff
@@ -186,19 +186,18 @@ def _handle_erratum(repo, type_id, unit_key, metadata, file_path, conduit, confi
     unit_data.update(metadata or {})
     unit_data.update(unit_key or {})
 
-    existing_unit = model_class.objects.filter(**unit_key).first()
-    new_unit = model_class(**unit_data)
+    unit = model_class(**unit_data)
+    unit._repo_id = repo.repo_id
```

Comment:
@mhrivnak 
tl;dr: It is not stored in db and it is a way to know repo_id during merge of pkglist.

We had one pkglist (which consisted of multiple collections) for one erratum and repo_id was stored within collection as you said. Now for one erratum we can have multiple pkglists (which still can consist of multiple collections) and we have repo_id (field in ErratumPkglist model) per pkglist so we no longer need repo_id to be stored within each collection.

Because of that change now we have to find out somehow a repo_id for erratum which is about to be synced/uploaded to use it during merge of pkglists [here in pre_save_signal](https://github.com/pulp/pulp_rpm/pull/1049/files#diff-df55e4e5c922fdb2a0fefedecd7013c5R1151).
The mongoengine version we use does not support passing arguments to the signals (e.g.  `model.save(**kwargs)`), implementing or backporting it seems to me like overkill. Any other ideas?

Does it make sense at all?